app/actors/hyku_addons/actors/member_collection_from_admin_set_actor.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  module Actors
+    class MemberCollectionFromAdminSetActor < Hyrax::Actors::AbstractActor
+      def create(env)
+        ensure_collection!(env) if enabled? && correct_permissions?(env)
+
+        next_actor.create(env)
+      end
+
+      # We shouldn't be doing anything on update as the collection will be assigned on create
+      delegate :update, to: :next_actor
+
+      protected
+
+        def ensure_collection!(env)
+          admin_set_id = env.attributes.dig("admin_set_id")
+
+          return unless admin_set_id.present?
+
+          title = AdminSet.find(admin_set_id).title.first
```

Comment:
`find` could throw an exception here if it can't find the record with that admin_set_id. Would that be possible on the workflow?

Reply:
So, because the admin_set is only set from the available options, i think this is ok for now. If the user messes with the URL params, thats on them.